### ISSUE TITLE
UV images and thumbnails not loading in production

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-lux#679]
As of about 9 AM this morning, I'm seeing no thumbnails in search results except for our Login required placeholders. 

IIIF manifests also aren't loading images - I spot checked across various collections including works that I've visited in the last couple of days.

Collection banner images are loading successfully.

UPDATE: 
At the same time this was occurring Curate prod was not responding, which probably caused IIIF requests to not respond. Curate came back on its own, but it was inaccessible from about 9:10 AM - 9:25 AM 8/5.

**Comment by SolomonHD** [comment]
Only thing I'm seeing so far is that the number of httpd processes is spiking around the time of the non-responsiveness. (13:20 UTC is 9:20 EST)
![image.png](https://images.zenhubusercontent.com/5c191329a896da5d83d69955/2a1543aa-f110-4d20-ad63-6e0cea8b72d3)

**Comment by SolomonHD** [comment]
Current plan is to bring down the PassengerMaxQueSize variable, currently set to 1000, down to 200-300, this should cause the server to respond with a 503/Server too busy message instead of appearing to be down. Hopefully this will increase clarity as to what's going on with the server.

Another thread is to change from prefork to worker MPM, this should allow for better performance. Worker MPM will need to be tested out to ensure compatibility.

**Comment by eporter23** [comment]
@SolomonHD some updates from testing in arch: I was able to trigger the same behavior (to a lesser degree) by:

1. Opening up and editing 5 or 6 large book objects in Curate in quick succession, keeping many tabs open and refreshing them. The editing step will cause the manifest to regenerate.
2. I occasionally got a blank screen in Curate in some tabs with the browser reload bar moving very slowly
3. I then went into digital-arch and saw that thumbnails were not loading, and UV images were not loading. After a minute or so (as the Curate edits died down), images came back.
